translation.json

```diff
@@ -0,0 +1,83 @@
+{
+    "widgets": {
+        "formBuilder": {
+            "dataSource": {
+                "loading": "Loading...",
+                "sending": {
+                    "requiredFields": "* Required fields"
+                },
+                "email": {
+                    "invalid": "The input is not a valid email address."
+                },
+                "file": {
+                    "uploaded": "Uploaded:",
+                    "uploadMessage": "To be uploaded",
+                    "instruction": "Choose file"
+                },
+                "image": {
+                    "instruction": "Choose image",
+                    "invalid": "The uploaded file is not a valid image. Please try again.",
+                    "actionLabels": {
+                        "takePhoto": "Take Photo",
+                        "choosePhoto": "Choose Existing Photo",
+                        "cancel": "Cancel"
+                    },
+                    "confirmMessage": "How do you want to choose your image?",
+                    "confirmLabel": "Choose Image",
+                    "required": "Please fill in required fields."
+                },
+                "interface": {
+                    "hidden": "Hidden"
+                },
+                "number": {
+                    "invalidLength": "This field only accept up to 15 digits.",
+                    "onlyPositiveDigitsAllowed": "Only positive digits are allowed.",
+                    "onlyIntegerDigitsAllowed": "Only integer digits are allowed.",
+                    "invalidDecimal": "Only digits or {{decimals}} digits after point are allowed."
+                },
+                "paragraph": {
+                    "defaultValue": "Paragraph"
+                },
+                "password": {
+                    "confirmPassword": "Confirm password",
+                    "mismatch": "Password confirmation does not match password.",
+                    "placeholder": "A password will be automatically generated"
+                },
+                "checkbox": {
+                    "defaultOptions": {
+                        "option1": "Option 1",
+                        "option2": "Option 2"
+                    }
+                },
+                "radio": {
+                    "defaultOptions": {
+                        "option1": "Option 1",
+                        "option2": "Option 2"
+                    }
+                },
+                "select": {
+                    "name": "Dropdown (single-select)",
+                    "placeholder": "-- Select one",
```

Comment:
The two dashes should not be part of the string

Reply:
Agreed